scripts/qiita-private-launcher

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
What is this private launcher script used for? Registering plugins on test environments?